src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -99,6 +110,9 @@ const PaymentAgentWithdraw = (() => {
                     // error handling
                     $agent_error.setVisibility(1);
                 }
+                if ($(field_ids.txt_amount).val()) {
+                    Validation.validate(form_id);
```

Comment:
can we put this in a function and call it both on line 131 and 113? since it's the same if condition and function call.

Reply:
Done!